std/range/package.d

```diff
@@ -7250,7 +7250,7 @@ if (isInputRange!Range)
     }
 
     // Assertion only.
-    private void dbgVerifySorted()
+    private auto dbgVerifySorted()
```

Comment:
The type is inferred as `void`.  It should probably be turned into a template function instead.
`private void dbgVerifySorted()()`